consumer/rs-g2l100/hardware-docs/README.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Hardware Documentation for boardname
+permalink: /documentation/consumer/boardname/hardware-docs/
```

Comment:
boardname should be changed to rs-g2l100

Reply:
> boardname should be changed to rs-g2l100

now, i fixed the problem, but it still has the error, I don't know what's causing it

Reply:
> now, i fixed the problem, but it still has the error, I don't know what's causing it

It is an error on the main website, by the look of it. The link checker is reporting a bad link in "index.html".

I can approve and force through this particular pull request but it may not build if there is a problem with the index page.